.docs/user-guide/usage.md

```diff
@@ -0,0 +1,210 @@
+# Usage
+Report status, Initiate storage operations
+
+---
+
+## Summary
+General format:
+
+```sh
+$ rexray [option flags] [command]
+```
+
+Get general or command specific help
+
+```sh
+$ rexray --help
+```
+
+```sh
+$ rexray [command] --help
+```
+
+## Global Option Flags
+These can be invoked with all commands
+
+| Flag | Alternate form | Description |
+| --- | --- | --- |
+| `-c` | `--config=""` | Path to a custom REX-Ray configuration file |
+| `-h` | `--host=""` | designate libStorage host to execute command |
+| `-l` | `--logLevel=""` | error, warn, info, debug |
+| `-s` | `--service=""` | designate specific libStorage service command pertains to |
+| `-v` | `--verbose=true` | emit verbose information to console |
+
+## Commands
+| Command | Description |
+| --- | --- |
+| version | return REX-Ray and libStorage versions |
+| env | return effective current configuration, in form normalized as environment variables |
+| adapter | return list of configured services, or list of host instances manageable by a service |
+| device | list or operate (mount,unmount,format) on OS devices |
+| service | manage REX-Ray running as an OS service |
+| volume | list or operate (create,remove,attach,detach,mount,unmount) on volumes |
+| help | return list of commands and global option flags |
+
+## Examples
+**Display Version**
+
+```shell
+$ rexray version
+REX-Ray
+-------
+Binary: /usr/bin/rexray
+Flavor: client+agent+controller
+SemVer: 0.8.1
+OsArch: Linux-x86_64
+Branch: v0.8.1
+Commit: 30e9082dd9917f0d218ca981f886d701110ce5f5
+Formed: Sat, 25 Feb 2017 03:00:28 UTC
+
+libStorage
+----------
+SemVer: 0.5.1
+OsArch: Linux-x86_64
+Branch: v0.8.1
+Commit: 35c7b6d96d5f17aa0c0379924615ae22c1ad3d45
+Formed: Sat, 25 Feb 2017 02:59:00 UTC
+```
+
+**Start REX-Ray Service**
+
+```shell
+$ sudo rexray service start
+Starting REX-Ray...SUCCESS!
+
+  The REX-Ray daemon is now running at PID 1455. To
+  shutdown the daemon execute the following command:
+
+    sudo /usr/bin/rexray stop
+```
+
+**List Volumes**
+
+```shell
+$ rexray volume ls
+ID                     Name       Status       Size
+vol-03ff12d1be6e8a65d             attached     8
+vol-549bd6d4                      unavailable  20
+vol-867e4906           mesos1-sw  available    16
+```
+
+| Status | Description |
+| --- | --- |
+| `attached` | volume is currently attached to an instance |
```

Comment:
Not *an* instance, it is attached to the *this* instance -- the one making the request to the libStorage API.